kratos/python/add_model_part_to_python.cpp

```diff
@@ -878,17 +890,17 @@ void AddModelPartToPython(pybind11::module& m)
         .def("AddNodalSolutionStepVariable", AddNodalSolutionStepVariable<bool>)
         .def("AddNodalSolutionStepVariable", AddNodalSolutionStepVariable<int>)
         .def("AddNodalSolutionStepVariable", AddNodalSolutionStepVariable<double>)
-        .def("AddNodalSolutionStepVariable", AddNodalSolutionStepVariable<array_1d<double, 3 > >)
+        .def("AddNodalSolutionStepVariable", AddNodalSolutionStepVariable<array_1d<double, 3>>)
```

Comment:
you are mixing clean up and actual changes..., but i will approve

Reply:
These were automatically added when saving the file. Deactivate the whitespaces.